runtime-testsuite/pom.xml

```diff
@@ -66,7 +66,7 @@
 		<dependency>
 		    <groupId>org.openjdk.jol</groupId>
 		    <artifactId>jol-core</artifactId>
-		    <version>0.16</version>
+		    <version>0.17</version>
```

Comment:
Is this updrade needed? `0.16` isn't compatible with with JDK 17, am I right? If so, please put these details into the commit message.

Reply:
I think this one is a typo